package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-mbaas-api",
-  "version": "6.1.5",
+  "version": "6.1.6",
```

Comment:
@sedroche thanks, I am not familiar with what needs to be done along with verison bump. Will update